worlds/Files.py

```diff
@@ -355,6 +356,8 @@ def write_token(self, token_type: APTokenTypes, offset: int, data: Union[bytes,
         """
         Stores a token to be used by patching.
         """
+        if not getattr(self, "tokens", None):
+            self.tokens = list()
```

Comment:
Getattr is kinda a red flag for me because it means that the object does not match the typing (i.e. the member is missing and it could crash elsewhere without pypi screaming), but assuming we keep it here, because mixins are special, which is probably fine, I'd still suggest to do
```suggestion
            self.tokens = []
```